main-src/processQueue.cjs

```diff
@@ -207,7 +229,7 @@ async function _processVideo(video, tmpDir) {
   if (PATH_TO_MODELS) {
     demucsExeArgs.push('--repo', PATH_TO_MODELS)
   }
-  await spawnAndWait(tmpDir, DEMUCS_EXE_NAME, demucsExeArgs)
+  await spawnAndWait(video.videoId, tmpDir, DEMUCS_EXE_NAME, demucsExeArgs)
```

Comment:
this feels a little wonky having to pass the video ID all the way down like this, what do you think?